packages/nimble-components/src/select/index.ts

```diff
@@ -853,6 +864,9 @@ export class Select
             this.filterInput.value = '';
         }
 
+        if (this.filterMode !== FilterMode.none) {
+            this.emitFilterInputEvent();
+        }
```

Comment:
Can this check be merged with the check above it? Or is there some subtle difference in these conditions?
```ts
if (this.filterInput) {...}
if (this.filterMode !== FilterMode.none) {...}
```
Aren't those the same check?